.github/DEVELOPMENT.md

```diff
@@ -215,6 +215,76 @@ public static int foo = 2130771968;
 
 5. Restore the commented code in `Directory.Build.targets`.
 
+## What branch should I use?
+
+- net7.0
+  - I want to use the net7.0 sdk and make changes that will be released with the .NET 7 release of MAUI
+- net6.0
+  - This PR seems like it should go out with a net6.0 service release
+- main (start here if you don't know what to use)
+  - I want to use the net6.0 sdk and make changes that will be released with the .NET7 release of MAUI
```

Comment:
Should this info be added to the [default PR template](https://github.com/dotnet/maui/blob/main/.github/PULL_REQUEST_TEMPLATE.md)?

Reply:
Good thought!
https://github.com/dotnet/maui/pull/8420